src/pal/src/misc/miscpalapi.cpp

```diff
@@ -454,3 +454,159 @@ EnsureOpenSslInitialized()
     return dwRet;
 }
 
+/*++
```

Comment:
I'd done that initially, but ran into issues where they weren't ending up in the libcoreclr binary, since nothing else in libcoreclr was using them and the linker was throwing them away, as far as I could tell.  Is there a good way to avoid that?